src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc

```diff
@@ -0,0 +1,39 @@
+---
+title: The Developer Workspace Button
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/the-developer-workspace-button
+summary:
+---
+
+:page-liquid:
+:parent-context-of-the-developer-workspace-button: {context}
+
+[id="the-developer-workspace-button_{context}"]
+= The Developer Workspace Button
```

Comment:
I think this title is only clear if you already know what you want -- maybe something like `Adding a markdown button for opening your repo in Che`?